docs/source/conf.py

```diff
@@ -98,7 +99,8 @@
 
 # General information about the project.
 project = "PyMC"
-copyright = "2021, The PyMC Development Team"
+year = date.today().strftime("%Y")
+copyright = f"{year}, The PyMC Development Team"
```

Comment:
This would mean we renounce the copyright of the content created before that date. If you want to be more technically correct we can change it to "2021-present, PyMC development team", probably even an earlier year. I have updated many dates in the past but have since learned that is actually wrong

Reply:
Thanks, fixed